pkg/cli/run.go

```diff
@@ -271,7 +271,10 @@ func (s *Run) Run(cmd *cobra.Command, args []string) error {
 		if s.Interactive {
 			return fmt.Errorf("cannot use --update/-u or --replace/-r with --dev/-i")
 		}
-		return updateHelper(cmd, args, s, c, isDir, cwd)
+		//if app does not exist but --update/--replace flag is used create app instead
+		if existingApp != nil {
```

Comment:
I'm a bit confused...should this be `if existingApp == nil`?

Reply:
no, if an existing app exists with the name we want to continue with the update. if existingApp==nil then we want to start an app with the provided name